wordle-web/src/views/WordleView.vue

```diff
@@ -9,6 +12,42 @@
 
   <h2>{{ guess }}</h2>
   <h3>{{ game.secretWord }}</h3>
+
+  <v-overlay :model-value="overlay" class="align-center justify-center" persistent>
+    <v-progress-circular color="primary" indeterminate size="64" />
+  </v-overlay>
+
+  <template>
+    <v-dialog v-model="nameDialog" width="500px" persistent>
+      <v-card height="200px" width="500px" title="Name:">
+        <v-text-field label="UserName" @input="nameUpdater($event.target.value)"></v-text-field>
+        <v-btn @click="nameDialog = false">Confirm Name</v-btn>
+      </v-card>
+    </v-dialog>
+  </template>
```

Comment:
I think it would be nice if the name dialog displayed right when someone went to play the game. It seemed like the requirements were suggesting that, but I could be wrong.

Reply:
I was thinking it meant to ask the user for their name towards the end, but I think your right.  I'll get that change implemented.